daffodil-core/src/test/scala/org/apache/daffodil/core/api/TestDsomCompiler3.scala

```diff
@@ -73,7 +73,7 @@ class TestDsomCompiler3 {
       // Verify things still work using specified tmpDir
       //
       val Seq(schema) = sset.schemas
-      val Seq(schemaDoc, _) = schema.schemaDocuments
+      val schemaDoc = schema.schemaDocuments.head
```

Comment:
For other reviewers, this line has been simplified to get the first schemaDoc in a more straightforward way.

Reply:
Yep, because there is now an extra include (GeneralFormatBase.xsd) there are now three docs in the `schema.schemaDocuments` list in a lot of tet. The first is still the root schema that we actually care about, but the pattern match fails. I figure switching to using `.head` will allow us to change the includes again if needed without breaking these tests, for example if IBM DFDL and Daffodil fix compatibility issues and the portable stuff goes away.